src/components/UserProfiles/UserProfileBase/__test__/UserProfileBase.test.tsx

```diff
@@ -1,28 +1,28 @@
-import UserProfileBase, { UserProfileBaseProps } from "..";
-
-import { render, screen } from "@testing-library/react";
-
-describe("<UserProfileBase />", () => {
-    let props: UserProfileBaseProps;
-
-    it("should exist children with default style", () => {
-        render(<UserProfileBase {...props} />);
-        expect(screen.getByText("Compose")).toBeInTheDocument();
-    });
-
-    it("should exist children with variant left", () => {
-        props = {
-            variant: "logged-in",
-        };
-        render(<UserProfileBase {...props} />);
-        expect(screen.getByText("Compose")).toBeInTheDocument();
-    });
-
-    it("should exist children with variant center", () => {
-        props = {
-            variant: "logged-out",
-        };
-        render(<UserProfileBase {...props} />);
-        expect(screen.getByText("Sign In")).toBeInTheDocument();
-    });
-});
+import UserProfileBase, { UserProfileBaseProps } from "..";
+
+import { render, screen } from "@testing-library/react";
+
+describe("<UserProfileBase />", () => {
+    let props: UserProfileBaseProps;
+
+    it("should exist children with default style", () => {
+        render(<UserProfileBase {...props} />);
+        expect(screen.getByText("Compose")).toBeInTheDocument();
+    });
+
+    it("should exist children with variant left", () => {
+        props = {
+            variant: "logged-in",
+        };
+        render(<UserProfileBase {...props} />);
+        expect(screen.getByText("Compose")).toBeInTheDocument();
+    });
+
+    it("should exist children with variant center", () => {
+        props = {
+            variant: "logged-out",
+        };
+        render(<UserProfileBase {...props} />);
+        expect(screen.getByText("Sign In")).toBeInTheDocument();
+    });
+});
```

Comment:
![image](https://user-images.githubusercontent.com/26225039/140066354-aa78d0a2-aa25-42a1-995f-c37849864e5c.png)
Because I don't think we're testing for the theme style

Reply:
My bad!